libs/ledgerjs/packages/types-live/src/account.ts

```diff
@@ -230,6 +233,7 @@ export type ChildAccountRaw = {
   id: string;
   name: string;
   starred?: boolean;
+  hidden?: boolean;
```

Comment:
are we sure we want to go that way? isn't it duplicating the current approach of "hidding" token of the system called "blacklist token ids"? We already have a way to hide token and it's via this env and via retriggering a synchronisation: that way, we were just REMOVING the accounts out of tokenAccounts instead of having a boolean to check everywhere (which is costly, it can be annoying to always have to filter these out, all the code that do `subAccounts.length` would also be badly impacted (eg to condition presence of sub accounts))

Reply:
Nice tips @gre I wasn't aware of the `blacklistedTokenIds` yet.
The issue I noticed here is that there are 2 reducers triggered for 1 redux action (settings and account).
The action in the settings reducer is correct but not in the account reducer (there are no SHOW_TOKEN action): https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-mobile/src/reducers/accounts.ts#L111.
An other issue is that when the `withoutToken` function is called we remove the token from from the list of subAccounts. The idea was kind of similar to the blacklistedTokenIds meaning that instead of removing the subAccount from memory we simply give it a property to make shown or hidden.
If it's possible I could try to refactor the `withoutToken` and `enableToken` functions to actually add/remove the tokens from this list.